### PR TITLE
routes for volunteer page

### PR DESCRIPTION
**Pull request by Artsyy**

### CODE REVIEW

File: HackMerced/2017F/server.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    path: '/volunteer',
+    handler: function (request, reply) {
+        reply('Hello, volunteer!');
+    }
+});
+
+//should probably make this get post as this should be a form volunteer fills out
+server.route({
+    method: 'GET',
+    path: '/volunteer/confirmation',
```
Comment (zzmarkzz321) [inline HackMerced/2017F/server.js:25]: No keep this as a Get

We might want to separate the routes for 
1. Submitting a form
2. Redirect to a confirmation

**Review by zzmarkzz321 — CHANGES_REQUESTED** [review]
Let's keep changes in this one PR. We do not have an explicit way to toggle these routes on our off in production, so we'll need all requirements included in one PR.

File: HackMerced/2017F/server.js (hunk 1 above)
Comment (zzmarkzz321) [reply HackMerced/2017F/server.js:25]: Unless we redirect to the home page and display a message/alert to let them know it has successfully been submitted

Comment (4shub) [reply HackMerced/2017F/server.js:25]: next time please more descriptive branch names

Comment (4shub) [reply HackMerced/2017F/server.js:25]: this is not how routing works BTW

Comment (zzmarkzz321) [reply HackMerced/2017F/server.js:25]: Redirection after form submission?

Comment (4shub) [reply HackMerced/2017F/server.js:25]: please look here for all you need
https://github.com/HackMerced/HackMerced/tree/master/HackMerced/2017F/src/server/src

**Pull request closed without merging**